ansible/lifecycle.yml

```diff
@@ -68,4 +75,74 @@
                   {% for instance in r_instances.instances %}
                   {{ "%-60s" | format(instance.tags.Name) }} {{ "%-10s" | format(instance.state.name) }} {{ instance.instance_type }}
                   {% endfor %}
+    - when: cloud_provider == 'osp'
+      environment:
+        OS_AUTH_URL: "{{ osp_auth_url }}"
+        OS_USERNAME: "{{ osp_auth_username }}"
+        OS_PASSWORD: "{{ osp_auth_password }}"
+        OS_PROJECT_NAME: "{{ osp_project_name }}"
+        OS_PROJECT_DOMAIN_ID: "{{ osp_auth_project_domain }}"
+        OS_USER_DOMAIN_NAME: "{{ osp_auth_user_domain }}"
+      block:
+        - when: ACTION == 'stop'
+          block:
+            - name: Gather instance facts
+              os_server_info:
+                filters:
+                  metadata:
+                    Project: "{{ project_tag }}"
```

Comment:
I think this key currently gets set lowercase when instances are created. In case it is case sensitive, either change the filter here or we need to change the meta we add to match.

I prefer all lowercase to avoid issues like this.

Reply:
In AWS tags, it's `Project` :/

I'll change it to use `guid` and `env_type` like i did later.